source/_patterns/02-organisms/50-borrower-dashboard/05-auto-repayments.mustache

```diff
@@ -31,14 +31,17 @@
 				<div style="float: right; margin-left: 1rem;">
 					<a class="auto-repayment-pause">Pause</a>
 				</div>
-				{{#nextRepayment}}
-					Next payment of <b>${{amount}}</b> will be automatically
-					repaid on <b>{{date}}</b>
-				{{/nextRepayment}}
 				{{#preDisbursal}}
 					Enrolled in automatic repayments. Your first repayment
 					will be scheduled once your loan funds are sent to you.
 				{{/preDisbursal}}
+				{{#nextScheduledAutoRepayment}}
+					Next payment of <b>${{amount}}</b> will be automatically
+					repaid on <b>{{date}}</b>
+				{{/nextScheduledAutoRepayment}}
+				{{^nextScheduledAutoRepayment}}
+					Please contact borrowers@kiva.org to get on a payment plan.
```

Comment:
This is placeholder (and below on the main dashboard) until we get to ARP-51.

Reply:
duely noted.